website/src/components/CRONComponents/CRONSummary/CronSummary.jsx

```diff
@@ -14,48 +14,53 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React from 'react';
-import { ResponsiveLine } from '@nivo/line'
-
-import '../CRONSummary/cronSummary.css'
-
-const CronSummary = ({data, setBenchmarktype, setIsLoadingChart, benchmarkType}) => {
-
-    const transformedData = [
-        {
-          id: 'QPSTotal',
-          data: data.Data.map((item) => ({
-            x: item.CreatedAt,
-            y: item.QPSTotal,
-          })),
-        },
-      ];
-
-      const getBenchmarkType = () => {
-        setBenchmarktype(data.Name)
-        
-      }
-
-    return (
-        <div className='cronSummary flex--column' onClick={getBenchmarkType}>
-            <div className='cronSummary__chart'>
-                <ResponsiveLine
-                    data={transformedData}
-                    width={300}
-                    height={100}
-                    enableGridX={false}
-                    enableGridY={false}
-                    colors={['#E77002']}
-                    />
-            </div>
-            <figure className='cronSummary__line'></figure>
-            <div className='cronSummary__text'>
-                <span>QPS Total</span>
-                <h3>{data.Name}</h3>
-                <i className="fa-solid fa-arrow-right cron--fa-arrow-right"></i>
-            </div>
+import React from "react";
+import { ResponsiveLine } from "@nivo/line";
+
+import "../CRONSummary/cronSummary.css";
+
+const CronSummary = ({ data, setBenchmarktype }) => {
+  const orange = "#E77002";
+
+  const transformedData = [];
+
+  if (data == null) {
+    transformedData.push({
+      id: "QPSTotal",
+      data: data.Data.map((item) => ({
+        x: item.CreatedAt,
+        y: item.QPSTotal,
+      })),
+    });
+  }
```

Comment:
Mhm this seems odd to me. Shouldn't we check for data not null? Here, if data is null then we attempt to do a `.Data.map` on it